### PR TITLE
Fix division by zero warning

### PR DESCRIPTION
**Pull request by zakariaejaidi**
PHP Warning:  Division by zero in /vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Calculation/Financial/CashFlow/Variable/Periodic.php on line 155

This is:

```
- [ x] a bugfix
- [ ] a new feature
- [ ] refactoring
- [ ] additional unit tests
```

Checklist:

- [ ] Changes are covered by unit tests
  - [ ] Changes are covered by existing unit tests
  - [ ] New unit tests have been added
- [x] Code style is respected
- [ ] Commit message explains **why** the change is made (see https://github.com/erlang/otp/wiki/Writing-good-commit-messages)
- [ ] CHANGELOG.md contains a short summary of the change
- [ ] Documentation is updated as necessary

### CODE REVIEW

**Comment by oleibman** [comment]
Do you have a case where this change is needed? If so, please add a unit test to demonstrate its effectiveness?

**Comment by MarkBaker** [comment]
When we implement a bugfix, we always try to create a unit test to verify the original problem and confirm that the bugfix works as expected. Would you be able to create a unit test for this fix?

File: src/PhpSpreadsheet/Calculation/Financial/CashFlow/Variable/Periodic.php [hunk 1]
```diff
@@ -160,7 +160,8 @@ public static function presentValue($rate, ...$args)
         for ($i = 1; $i <= $countArgs; ++$i) {
             // Is it a numeric value?
             if (is_numeric($aArgs[$i - 1])) {
-                $returnValue += $aArgs[$i - 1] / (1 + $rate) ** $i;
+                $val = (1 + $rate) ** $i;
+                $returnValue += $val != 0 ? $aArgs[$i - 1] / $val : 0;
```
Comment (MarkBaker) [inline src/PhpSpreadsheet/Calculation/Financial/CashFlow/Variable/Periodic.php:164]: `$val` will always be a float, so why do a loose types comparison here?

**Review by MarkBaker — CHANGES_REQUESTED** [review]

**Comment by oleibman** [comment]
No update in over a year. Closing.

**Pull request closed without merging**